ErrorHandling.js

```diff
@@ -0,0 +1,59 @@
+const {
+    CarDamageException,
+    FileInsuranceClaimException,
+    InjuryException,
+    AirbagDeploymentFailedException,
+    GasLeakException
+} = require('./exceptions');
+
+const {
+    callMechanic,
+    provideMedicalAssistance,
+    checkAirbagDeployment,
+    handleGasLeak,
+    carDamageHandling,
+    insuranceClaimHandling,
+    AssistanceProvide,
+    airbagFailureCheck,
+    gasLeakHandling
+} = require('./functions');
+
+const SenerioEnum = {
+  isCarDamage: true,
+  isAnyCasuality: false,
+  isAirbagDeployed: false,
+  isGasLeak: false
+};
+
+try {
+     drive()
+} catch (error) {
+    if (error instanceof CarDamageException) {
+        callMechanic();
+    } else if (error instanceof FileInsuranceClaimException) {
+        fileInsuranceClaim();
+    } else if (error instanceof InjuryException) {
+        provideMedicalAssistance();
+    } else if (error instanceof AirbagDeploymentFailedException) {
+        checkAirbagDeployment();
+    } else if (error instanceof GasLeakException) {
+        handleGasLeak();
+        provideMedicalAssistance();
+    }
+} finally {
+    AssistanceProvide();
```

Comment:
Use Proper function in finally because it will run in every case irrespective of accident occur.

Reply:
sure I changed this